src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_functionapp_commands.py

```diff
@@ -508,6 +508,59 @@ def test_functionapp_consumption_linux_powershell(self, resource_group, storage_
         self.cmd('functionapp config show -g {} -n {}'.format(resource_group, functionapp_name), checks=[
             JMESPathCheck('linuxFxVersion', 'PowerShell|7.2')])
 
+
+class FunctionappDapr(LiveScenarioTest):
+    @AllowLargeResponse(8192)
+    @ResourceGroupPreparer(location="northeurope")
+    @StorageAccountPreparer()
+    def test_functionapp_dapr_config_e2e(self, resource_group, storage_account):
+        functionapp_name = self.create_random_name(
+            'functionappdapr', 24)
+        managed_environment_name = self.create_random_name(
+            'managedenvironment', 40
+        )
+
+        self.cmd('containerapp env create --name {} --resource-group {} --location {} --logs-destination none'.format(
+            managed_environment_name,
+            resource_group,
+            "northeurope"
+        ))
+
+        self.cmd('functionapp create -g {} -n {} -s {} --environment {} --dapr-app-id daprappid --dapr-app-port 800 --dhmrs 4 --dhrbs 50 --dapr-log-level debug --enable-dapr true --functions-version 4'.format(
+            resource_group,
+            functionapp_name,
+            storage_account,
+            managed_environment_name
+        )).assert_with_checks([
+            JMESPathCheck('daprConfig.enabled', True),
+            JMESPathCheck('daprConfig.appId', 'daprappid'),
+            JMESPathCheck('daprConfig.appPort', 800),
+            JMESPathCheck('daprConfig.httpReadBufferSize', 50),
+            JMESPathCheck('daprConfig.httpMaxRequestSize', 4),
+            JMESPathCheck('daprConfig.logLevel', 'debug'),
+            JMESPathCheck('daprConfig.enableApiLogging', False)
+        ])
+
+        time.sleep(1200)
+
+        self.cmd('functionapp config container set -g {} -n {} --dapr-app-id daprappid1 --dapr-app-port 80 --dal --dhmrs 6 --dhrbs 60 --dapr-log-level warn --enable-dapr false'.format(
+            resource_group,
+            functionapp_name
+        ))
+
+        time.sleep(1200)
```

Comment:
is this 1200 just be on safer side that the functionapp config command will be finished by then?

Reply:
Yes, otherwise, we get a 409 error since the previous operation is not done